traitsui/qt4/ui_base.py

```diff
@@ -106,26 +103,18 @@ class ButtonEditor(Editor):
     #-------------------------------------------------------------------------
 
     def __init__(self, **traits):
```

Comment:
Yes - the super-superclass's `__init__` method expects things which aren't available here.  I am not convinced that this should be a subclass of `Editor`, but I haven't sat down and worked out exactly what is going on here and it is orthogonal to this PR.